src/synchronization/stackImagePositionSynchronizer.js

```diff
@@ -22,7 +22,7 @@ export default function (synchronizer, sourceElement, targetElement) {
   let minDistance = Number.MAX_VALUE;
   let newImageIdIndex = -1;
 
-  external.$.each(stackData.imageIds, function (index, imageId) {
+  Array.prototype.forEach.call(stackData.imageIds, function (index, imageId) {
```

Comment:
why not just stackData.imageIds.forEach( ... ?

Reply:
Oh, and index / imageId are switched when going from jQuery to forEach. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

Reply:
I wasn't sure that imageIds was a real array, but if it's always a real array then it should be fine to use `.forEach` directly.

Fixing!

Reply:
Looked it up, and in both cases they are real arrays.